Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:xenial
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup
+RUN echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache
+
+RUN \
+  apt-get update && \
+  apt-get -y install \
+          software-properties-common
+
+RUN apt-get -y install python-pip git postgresql-server-dev-9.5 graphviz
```

Comment:
what's the reason for including `postgresql-server-dev-9.5` ? that's a pretty heavy dependency. (view towards chembl sql downloader?)